config/rollup.config.js

```diff
@@ -1,8 +1,13 @@
+import fs from 'fs';
 import nodeResolve from 'rollup-plugin-node-resolve';
 import babel from 'rollup-plugin-babel';
 import memory from 'rollup-plugin-memory';
 
+const pkg = JSON.parse(fs.readFileSync('./package.json'));
```

Comment:
Reviving, but strolling thru 🚶... 

```js
const pkg = require('../package');
```

will work perfectly fine -- I do it in all my modules. 😄